src/test/java/games/strategy/triplea/delegate/MoveValidatorTest.java

```diff
@@ -1,27 +1,40 @@
 package games.strategy.triplea.delegate;
 
+import static games.strategy.triplea.delegate.GameDataTestUtil.addTo;
+import static games.strategy.triplea.delegate.GameDataTestUtil.territory;
 import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
 import static org.junit.Assert.assertTrue;
 import static org.junit.Assert.fail;
 
 import java.util.ArrayList;
 import java.util.Collection;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
 
 import org.junit.Before;
 import org.junit.Test;
 
+import games.strategy.engine.data.GameData;
+import games.strategy.engine.data.PlayerID;
 import games.strategy.engine.data.Route;
 import games.strategy.engine.data.Territory;
 import games.strategy.engine.data.Unit;
 import games.strategy.triplea.TripleAUnit;
+import games.strategy.triplea.delegate.dataObjects.MoveValidationResult;
+import games.strategy.triplea.xml.TestMapGameData;
 import games.strategy.util.Match;
 
 public class MoveValidatorTest extends DelegateTest {
 
+  private GameData twwGameData;
```

Comment:
Before you renamed this field, I was going to suggest simply making it a local in the one test method that is using it.  Right now each test that runs is loading two maps.  If you add new test methods in the future that require the TWW map, that's probably the time to split out a separate fixture for these types of `MoveValidator` tests.

Reply:
Fair point. I'll move it into the new unit test for now.